addon/models/address.js

```diff
@@ -16,21 +37,30 @@ export default class Address extends XMLModel {
         street: String,
         houseNumber: Number,
         swissZipCode: Number,
+        swissZipCodeAddOn: Number,
+        addressLine2: String,
         town: String,
-        country: String,
+        country: Country,
       },
     });
   }
 
   static template = `
   <address>
+    {{#if (not (eq model.country.countryNameShort "ch"))}}
+      {{{modelField model "addressLine2" namespace="ns5:"}}}
+    {{/if}}
     <ns5:street>{{model.street}}</ns5:street>
     <ns5:houseNumber>{{model.houseNumber}}</ns5:houseNumber>
-    <ns5:town>{{model.town}}</ns5:town>
-    {{{modelField model "swissZipCode" namespace="ns5:"}}}
-    <ns5:country>
-      <ns5:countryNameShort>{{model.country}}</ns5:countryNameShort>
-    </ns5:country>
+    {{!--TODO: Town is ignored by the API, N/A is returned. 
+      However, an error is returned if it isn't present.
+      Remove once API no longer requires field --}}
+    <ns5:town></ns5:town>
```

Comment:
I thought we should only display town if its not `country == switzerland`?

Reply:
`town` is not displayed, only `addressLine2`